.github/CODEOWNERS

```diff
@@ -10,7 +10,7 @@
 /pybamm/models/ @brosaplanella @DrSOKane @rtimms @valentinsulzer @TomTranter @rtimms
 /pybamm/parameters/ @brosaplanella @DrSOKane @rtimms @valentinsulzer @TomTranter @rtimms  @kratman
 /pybamm/plotting/ @martinjrobins @rtimms @Saransh-cpp @valentinsulzer @rtimms  @kratman @agriyakhetarpal
-/pybamm/solvers/ @martinjrobins @rtimms @valentinsulzer @TomTranter @rtimms
+/pybamm/solvers/ @martinjrobins @rtimms @valentinsulzer @TomTranter @rtimms @MarcBerliner
```

Comment:
Not sure why, but github is marking this as an error. His username looks correct to me though

Reply:
![image](https://github.com/user-attachments/assets/ae52f081-d093-427a-9fe8-6e9d0a36c14a)

Reply:
Ah ok I'll close this until then